Build.pm

```diff
@@ -159,13 +159,13 @@ my %subst_defaults = (
     'apt-utils', 'cpio', 'dpkg-dev', 'live-build', 'lsb-release', 'tar',
   ],
   'system-packages:mkosi:rpm' => [
-    'mkosi',
+    'mkosi', 'createrepo', 'gzip',
   ],
   'system-packages:mkosi:deb' => [
-    'mkosi',
+    'mkosi', 'dpkg-dev', 'lsb-release', 'gzip',
```

Comment:
Same here, `dpkg-dev` isn't really useful to mkosi without having apt as well. I'm a bit confused what these deps are for since building debian/ubuntu/arch from a opensuse tumbleweed doesn't work at all because the necessary tools are not packaged in opensuse.

Reply:
As above, this is what will be pulled in for debian-based image builds and that the glue from this repo itself will use, the mkosi package can pull in what it needs by itself